vignettes/articles/symptom-surveys.Rmd

```diff
@@ -423,7 +423,7 @@ res_err4 <- res_all4 %>%
 knitr::kable(
   res_err4 %>%
     group_by(model, lead) %>%
-    summarize(err = median(err), n = length(unique(forecast_date))) %>%
+    summarise(err = median(err), n = length(unique(forecast_date))) %>%
```

Comment:
You don't like the Canadian spelling?

Reply:
truth be told, I use them 50/50, and couldn't tell you which one was :canada: XD

 I think this happened when I did a grep renaming because I was getting errors from my own use of `summarize`.